public/Leo Jayachandran/Glossarizer/libretextsGlossarizer.css

```diff
@@ -0,0 +1,87 @@
+.glossarizer_replaced:hover {
+    border-bottom: 1px #333 dotted;
+}
+
+.glossarizer_replaced {
+    cursor: help;
+    display: inline;
+    text-decoration-line: none;
+}
+
+.tooltip {
```

Comment:
`.tooltip` appears to be a very general classname and may already be in use. If possible, could you increase the complexity of this CSS query and/or modify the classname?

Reply:
I hadn't thought of that, I'll use a different classname

Reply:
classname has been changed to "glossarizerTooltip"